run/aix2john.pl

```diff
@@ -1,37 +1,57 @@
 #!/usr/bin/env perl
-#/
-#  This software is Copyright (c) 2013 Konrads Smelkovs <konrads.smelkovs@kpmg.co.uk>,
-#  and it is hereby released to the general public under the following terms:
-#  Redistribution and use in source and binary forms, with or without
-#  modification, are permitted.
```

Comment:
This used to have a copyright statement and license. Now it does not. Please add your copyright statement (only yours if you kept nothing from the original file) and keep the license statement intact.